lib/resin-lint.ts

```diff
@@ -223,8 +229,12 @@ export const lint = function(passedParams: any) {
 		})
 			.then(function() {
 				let configOverridePath;
-				const resinLintConfiguration = options.argv.typescript
-					? configurations.typescript
+				const prettierCheck = options.argv.prettier !== false;
+				const typescriptCheck = options.argv.typescript;
+				const resinLintConfiguration = typescriptCheck
+					? prettierCheck
```

Comment:
Let's also bump prettier to the latest version to get better ternary formatting.

Reply:
`npm outdated` already marks it as up to date

Reply:
Oh sorry, I expected to see flattened ternaries as [per v1.15](https://github.com/prettier/prettier/pull/4767#issuecomment-401764876) but this a totally different case.